src/aws_secretsmanager_caching/config.py

```diff
@@ -54,7 +53,7 @@ class SecretCacheConfig:
         "exception_retry_growth_factor": 2,
         "exception_retry_delay_max": 3600,
         "default_version_stage": "AWSCURRENT",
-        "secret_refresh_interval": timedelta(hours=1).total_seconds(),
+        "secret_refresh_interval": 3600,
```

Comment:
Is this just a style thing? Or does `timedelta()` no longer work with the new updates?

Reply:
timedelta returns a float, but that argument is needed to be an int and since that line will always return the same value I substituted it's int value

Reply:
Why dont we just wrap it in int()? I think its easier to read int(timedelta(hours=1).total_seconds()) than just setting it to 3600

Reply:
I figure it's more tonally consistent with the rest of the defaults and cleaner because that line of code will always return 3600.0

Reply:
I could add inline comments next to them describing the units/meaning if that would help

Reply:
im ok with this. its defined in the doc above the config and i just missed it